App.js

```diff
@@ -8,23 +8,31 @@
 
 import React from 'react';
```

Comment:
The file does not exist on master, Test should be added somewhere under `./src/*`